dev/ansible/02_run_terraform.yml

```diff
@@ -1,8 +1,23 @@
+- name: Download Terraform Latest
+  become: yes
+  get_url:
+      url: https://releases.hashicorp.com/terraform/0.13.3/terraform_0.13.3_linux_amd64.zip
+      dest: /tmp
+      mode: 0755
+
+- name: Unarchive a file that is already on the remote machine
+  become: yes
+  unarchive:
+      src: /tmp/terraform_0.13.3_linux_amd64.zip
+      dest: /usr/local/bin
+      remote_src: yes
+
```

Comment:
Oh, I forgot to remove this from the PR. I don't think we need this anymore? @dangernil

Reply:
You're right